### ISSUE TITLE
Verify all English translations

### ISSUE THREAD

**Issue by jbrooksuk** [CachetHQ/Cachet#2982]
There are a couple of places where I think the English translations could be improved. We need to:

- Spell check
- Grammar check
- Be more informative where needed, sometimes we don't explain settings or what's going on clearly
- Be concise, there are a few strings which are way too wordy

This will likely leave a lot of translations with some empty strings, but I'd rather that then an unclear application.

**Comment by jbrooksuk** [comment]
Pinning this issue. I think we should do this before we finish work on implementing the new design.

**Comment by BaileyJM02** [comment]
Who's able to verify translations?

**Comment by BaileyJM02** [comment]
I also just went though all of the files under 2.4, it looks about ready apart from 2 or 3 issues now.

**Comment by jbrooksuk** [comment]
@BaileyJM02 myself and others. I'll take a look at any suggestions shortly.

**Comment by jbrooksuk** [comment]
Thanks for doing this!

I'm going to close this issue now anyway.
